### ISSUE TITLE
[Bug] EmailRep

### ISSUE THREAD

**Issue by KRUXLEX** [TheHive-Project/Cortex-Analyzers#750]
**Describe the bug**
EmailRep doesn't support a APIKey. It isn't necessary but for free we have bigger limit of request

**To Reproduce**
Non

**Expected behavior**
Add this to script:
```
self.emailrep = self.get_param('config.key', None, 'Missing EmailRep API key') //emailrep = EmailRep('<your api key>')
``` 

**Complementary information**
documentation https://github.com/sublime-security/emailrep.io-python

**Work environment**
Doesn't necessery

**Possible solutions**
Fix

**Additional context**
nOpe.

**Comment by dadokkio** [comment]
Hi @KRUXLEX can you please check if the pull resolves the issue?

**Comment by KRUXLEX** [comment]
I will add it tomorrow and give you feedback

**Comment by dadokkio** [comment]
thanks, also because I asked a key yesterday but I didn't receive any reply yet.. 
so I've written random code to test it :smiley:

**Comment by KRUXLEX** [comment]
`__init__() takes 1 positional argument but 2 were given`

**Comment by dadokkio** [comment]
have you removed the old emailrep.py file and installed the emailrep library?

**Comment by KRUXLEX** [comment]
forgot `pip3 install -r` :D checked... Working but in cortex don't show field for key input

**Comment by dadokkio** [comment]
Did you perform a refresh of all the analyzers in cortex?

**Comment by KRUXLEX** [comment]
Yea, I had problem with Docker. Fixed. I input ApiKey, I wil test that limit work. Give me time until tommorow.

**Comment by KRUXLEX** [comment]
Looks it working correctly. Dont know what is limit without api. But with cay it's start return 429 after 50 request.

**Comment by garanews** [comment]
fixed in 2.7.0
